scanpipe/pipes/d2d.py

```diff
@@ -235,10 +235,6 @@ def map_java_to_class(project, logger=None):
             )
         _map_java_to_class_resource(to_resource, from_resources, from_classes_index)
 
-    # Flag not mapped .class in to/ codebase
```

Comment:
@tdruez 

Do we want to update the status of unmapped/unmatched `.class` files after the purldb match?

Reply:
IMHO matching implies a status update of sorts.

Reply:
@pombredanne 

The status of Resources are updated when they have a successful match against purldb. (https://github.com/nexB/scancode.io/blob/class-file-match/scanpipe/pipes/d2d.py#L490)